sdk/options.go

```diff
@@ -151,3 +152,11 @@ func WithPlatformConfiguration(platformConfiguration PlatformConfiguration) Opti
 		c.platformConfiguration = platformConfiguration
 	}
 }
+
+// WithIPC returns an Option that indicates the SDK should use IPC for communication
+// this will allow the platform endpoint to be an empty string
+func WithIPC() Option {
```

Comment:
Suggestion - why not use a custom scheme, since this implies a different transport?

Reply:
The issue with this is that there is no scheme at all. When we bind the SDK into the platform, it's an empty string `sdk.New("")` I'm open to a new name... maybe `WithNoEndpoint()` or something like that?

Reply:
I'm fine with this since we intend to support it indefinitely